public/js/auth.js

```diff
@@ -3,13 +3,11 @@ function send_infos() {
   var pwd = document.getElementById("password").value;
 
   overload_xhr(
-    "GET",
-    "/api/user",
+    "POST",
+    "/api/user/login",
     function(xhr){
       token = JSON.parse(xhr.response)['token'];
-      username = JSON.parse(xhr.response)['user'];
-      sessionStorage.setItem('username',username);
-      sessionStorage.setItem('token',token);
+      sessionStorage.setItem('Token',token);
```

Comment:
I didn't pay attention during the first review but the token should be stored in cookies not in session. So when you reopen your browser, the token is preserved as long as the expiration date is not reached. We can have the token in cache expires at the same moment as the browser cookie.
To manage Vanilla JS cookies, you can check this page : https://www.w3schools.com/js/js_cookies.asp